library/src/morpheus-master/sass/modules/tool/rubrics/_rubrics.scss

```diff
@@ -23,6 +23,14 @@
         @include display-flex;
         @include justify-content(space-between);
 
+        .linkStyle{
+            &:hover, &:link, &:visited{
+                text-decoration: none !important;
+                color: black !important;
+                padding: 4px !important;
```

Comment:
You should be using the `$standard-space` variable for adding a space.

Do you only need this padding on one side?